src/_xoConnection.js

```diff
@@ -123,21 +131,50 @@ class XoConnection extends Xo {
     return id;
   }
 
+  async createResources(resourcesToCreate, disposer) {
+    const resources = {};
+    for (const typeOfResources in resourcesToCreate) {
+      for (const resource in resourcesToCreate[typeOfResources]) {
+        const { id } = await this.call(
+          METHOD[typeOfResources].create,
+          resourcesToCreate[typeOfResources][resource]
+        );
+        disposer.push(METHOD[typeOfResources].delete, { id });
+        resources[typeOfResources] = {
+          ...resources[typeOfResources],
+          [resource]: id,
+        };
+      }
+    }
+    return resources;
+  }
+
+  async createDurableResources(resources) {
```

Comment:
This method is not needed.

Reply:
So, each time we create resource, it will be deleted at the end of all tests ?

Reply:
Yes, why not directly use  `createResources`?

Reply:
Because I thought resources may not be deleted at the same time. I will change that :+1: